turborepo-tests/helpers/test_example.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Start by figuring out which example we're testing and its package manager
+example_path=$1
+package_manager=$2
+
+# If you need to do some debugging, you can set these to true
+debug=false
+force=false
+
+# Use the right command for each package manager
+if [ "$package_manager" == "npm" ]; then
+  package_manager_command="npm install"
+  update_turbo_version="npm install turbo@latest -w"
+elif [ "$package_manager" == "pnpm" ]; then
+  package_manager_command="pnpm install"
+  update_turbo_version="pnpm install turbo@latest -w"
+elif [ "$package_manager" == "yarn" ]; then
+  package_manager_command="yarn"
+  update_turbo_version="yarn upgrade turbo@latest -W"
+fi
+
+# All examples implement these two tasks
+# and it's reasonable to assume that they will continue to do so
+turbo_command="turbo build lint --output-logs=errors-only"
+
+# Head into the example directory
+cd $example_path
```

Comment:
This will inhert `vercell/turbo/.git` index. That seems like something that won't end well. Do we need to isolate the git repo here?

Reply:
Yeahhh, this was something that I thought about, but I'm struggling to figure out a way it will hurt these tests in practice...

Reply:
Ah, okay, I see the setup_git.sh. No reason not to!

Reply:
You'll also have to clean it up after the test is done and make sure that it's cleaned up if the test fails. Leaving an orphan `.git` inside our existing git repo will turn it into a submodule of sorts, and mean that changes won't be pushed up correctly.